app.js

```diff
@@ -1,15 +1,31 @@
 const express = require('express');
 const app = express();
+const multer = require('multer');
+const path = require('path');
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
@iamikenna I think these lines won't be needed here anymore since you have move the code to `multer.js`. Can you check?